test/02LessonTwoTest.js

```diff
@@ -7,13 +7,23 @@ function whichSpecies(character) {
     // should return "cat" when character is 'garfield'
     // should return "fish" when character is 'nemo'
     // should return false if character is anything else
-
-}
+    if (character === 'scooby') {
+      return 'dog';
+    } else if (character === 'garfield') {
+      return 'cat';
+    } else if (character === 'nemo') {
+      return 'fish';
+    } else {
+      return "false";
+    }
 
 function isEven(number) {
     // should return true is number is even (divisible by 2)
-
-}
+    if (number % 2 === 0) {
+      console.log("I'm Even!");
+    } else {
+      console.log("I'm Not Even!");
+    }
 
```

Comment:
You're missing a closing curly brace for the `isEven()` function here, so this test doesn't run.

```javascript
function isEven(number) {
    // should return true is number is even (divisible by 2)
    if (number % 2 === 0) {
      console.log("I'm Even!");
    } else {
      console.log("I'm Not Even!");
    }
}
```

Also, this function needs to return **true** if **number** is even, **false** otherwise.